3.0/vector_tile.proto

```diff
@@ -69,6 +109,12 @@ message Tile {
                 // See https://github.com/mapbox/vector-tile-spec/issues/47
                 optional uint32 extent = 5 [ default = 4096 ];
 
+                repeated string string_values = 7;
+                repeated double double_values = 8 [ packed = true ];
+                repeated float float_values = 9 [ packed = true ];
+                repeated int64 int64_values = 10 [ packed = true ];
+                repeated uint64 uint64_values = 11 [ packed = true ];
```

Comment:
Please use `sint64` instead of `int64` since this would only be preferred over `uint64` when the value is negative.